packages/mobile/src/components/social-button/SocialButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useMemo } from 'react'
+
+import type { ButtonProps } from 'app/components/core'
+import { Button } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+export type SocialButtonProps = ButtonProps
+
+const useStyles = makeStyles(({ spacing }) => ({
+  button: {
+    padding: spacing(3),
+    height: 64
+  },
+  text: {
+    fontSize: 20
+  },
+  icon: {
+    height: 20,
+    width: 20,
+    marginRight: 12
+  }
+}))
+
+export const SocialButton = (props: SocialButtonProps) => {
+  const styles = useStyles()
+  const buttonStyles = useMemo(
```

Comment:
Fun, was this causing a lot of rerenders otherwise?

Reply:
No it's probably not necessary. I just pulled it over from the other file but I'm thinking I'll get rid of it